preview_utils.py

```diff
@@ -0,0 +1,157 @@
+import os
+import shutil
+import threading
+import time
+
+import sublime
+
+
+_ST3 = sublime.version() >= "3000"
+if _ST3:
+    from .latextools_utils import cache, get_setting
+
+
+_lt_settings = {}
+
+
+def plugin_loaded():
+    global _lt_settings
+    _lt_settings = sublime.load_settings("LaTeXTools.sublime-settings")
+
+
+def convert_installed():
+    """Return whether ImageMagick/onvert is available in the PATH."""
+    if hasattr(convert_installed, "result"):
+        return convert_installed.result
+
+    convert_installed.result = shutil.which("convert") is not None
```

Comment:
This won't work on ST2 since `shutil.which()` was added in Python 3.3. The fix is to import which from `latextools_utils.system` which is identical to `shutil.which()`.